timetracker/entries/migrations/0003_auto_20150726_0653.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.utils.timezone
+
+
+def retain_clients(apps, schema_editor):
+    Project = apps.get_model("entries", "Project")
+    Client = apps.get_model("entries", "Client")
+    for project in Project.objects.all():
+        # Use get_or_create to ensure that clients are duplicated
+        client, created = Client.objects.get_or_create(name=project.client)
+        if created:
+            client.save()
```

Comment:
The `get_or_create` function will save the object by default, so saving isn't required here.